### DISCUSSION TITLE
Severe flaw in developer profiles

### DISCUSSION THREAD

**Discussion by ghost** [joemasilotti/railsdevs.com#721] [Ideas]
There is a severe flaw in developer profiles use case:-

# Problem

The day, 13th Sep 2022, I signed up on Railsdevs.com I received 2 to 3 enquiries next day related to job/work BUT now it has been more than 40+ days I haven't received any new enquiry.

# Reasons

1. Developer profiles which are old or stale go down and down the developers list when new developer profiles are created
2. Hiring managers are lazy to do full search and just pick the ones which appear on the first page of the list or which appear on the homepage of railsdevs.com
![image](https://user-images.githubusercontent.com/111341412/198969365-4444fa61-7f04-4d76-9c5a-ea470bed23fb.png)

# Solution

There can be some sort of Random displaying of developers on both the pages 'Home' and 'Developers' so that to each hiring manager a different set of developers is shown.

**Comment by joemasilotti** [comment]
I agree!

Implementing a "random" sorting sounds like a great idea to get more folks to the top of the list.

I'd want to make sure that the results only change at most once per day, though.

Do you have any interest in implementing this?

**Comment by jbennett** [comment]
Hey Joe, I setup a PR with an initial stab at this #748. It's grabbing random developers ids and caches them for 1 day. let me know your thoughts.
